### PR TITLE
Added Device Vitals page, added new "tree" in Navigation Panel

### PR DESCRIPTION
**Pull request by aoleszyn**
### Description
Added Device Vitals page, added new "tree" in Navigation Panel

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### CODE REVIEW

File: docs/mobile-apps/features/device-vitals.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+id: device-vitals
+title: Device Vitals
+---
```
Comment (lysannep) [inline docs/mobile-apps/features/device-vitals.md:4]:
```suggestion
sidebar_label: Device Vitals
---
```

Comment (lysannep) [reply docs/mobile-apps/features/device-vitals.md:4]: As per our style guide, we should also include a 'sidebar_label' at the top of the docs page: https://docs.saucelabs.com/contributing/style-guide/#frontmatter

File: docs/mobile-apps/features/device-vitals.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+id: device-vitals
+title: Device Vitals
+sidebar_label: Device Vitals
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Device Vitals is a functionality that gives you the ability to debug and observe an app’s performance during test execution for both, [Live](/mobile-apps/live-testing/live-mobile-app-testing/) and [Automated](/mobile-apps/automated-testing/) testing. With this feature, the app's performance is recorded on our side during the test session and displayed for your test results.
```
Comment (kristofmuhi) [inline docs/mobile-apps/features/device-vitals.md:10]: > and observe an app’s performance

i would highlight:

_and observe a mobile app's performance_

File: docs/mobile-apps/features/device-vitals.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+2. [Upload your mobile app to Sauce Labs](/mobile-apps/app-storage/) through the UI or CLI.
+3. After you’ve uploaded your app, return to the **Live** > **Mobile App** page, hover your mouse over your app, then select **Settings**. <br/><img src={useBaseUrl('img/mobile-apps/networkcapturescr.png')} alt="Mobile app settings navigation" width="600"/>
+4. Under **Default Settings**, toggle **Instrumentation** on. 
+   <br/><img src={useBaseUrl('img/mobile-apps/Device-Vitals-screenshot-1.png')} alt="Mobile app settings navigation" width="600"/>
+
+Now you can start your live or automated testing session. Your device vitals will be captured and be part of your test results page. 
+
+## Accessing the App Performance metrics 
+
+The device vitals recording will be included in the test results page. You will be able to compare your results and analyze the app performance.
```
Comment (kristofmuhi) [inline docs/mobile-apps/features/device-vitals.md:32]: @aoleszyn can we add a screenshot please for the feature?

Comment (kristofmuhi) [reply docs/mobile-apps/features/device-vitals.md:32]: i can pair with you on this one. We should run a test with our demo app, and add screenshot how it looks like

File: docs/mobile-apps/features/device-vitals.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+App performance is enabled by default if you have the instrumentation feature enabled.
+If Instrumentation is disabled, follow the steps below to enable it: 
+
+1. On Sauce Labs, click **Live** > **Mobile App**.
+2. [Upload your mobile app to Sauce Labs](/mobile-apps/app-storage/) through the UI or CLI.
+3. After you’ve uploaded your app, return to the **Live** > **Mobile App** page, hover your mouse over your app, then select **Settings**. <br/><img src={useBaseUrl('img/mobile-apps/networkcapturescr.png')} alt="Mobile app settings navigation" width="600"/>
+4. Under **Default Settings**, toggle **Instrumentation** on. 
+   <br/><img src={useBaseUrl('img/mobile-apps/Device-Vitals-screenshot-1.png')} alt="Mobile app settings navigation" width="600"/>
+
+Now you can start your live or automated testing session. Your device vitals will be captured and be part of your test results page.
```
Comment (kristofmuhi) [inline docs/mobile-apps/features/device-vitals.md:28]: We should add here information about:
CPU performance, what do we capture. 
Memory performance. 

Differentiation between iOS/Android UI responsiveness

File: docs/mobile-apps/features/device-vitals.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
+---
+id: device-vitals
+title: Device Vitals
```
Comment (kristofmuhi) [inline docs/mobile-apps/features/device-vitals.md:3]: App Diagnostics

File: docs/mobile-apps/features/device-vitals.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+sidebar_label: Device Vitals
```
Comment (kristofmuhi) [inline docs/mobile-apps/features/device-vitals.md:4]: App Diagnostics

File: docs/mobile-apps/features/device-vitals.md (hunk 2 above)
Comment (lysannep) [inline docs/mobile-apps/features/device-vitals.md:10]:
```suggestion
Device Vitals is a functionality that allows you to debug and observe an app’s performance during test execution for both [Live](/mobile-apps/live-testing/live-mobile-app-testing/) and [Automated](/mobile-apps/automated-testing/) testing. With this feature, the app's performance is recorded on our side during the test session and displayed for your test results.
```

Comment (aoleszyn) [reply docs/mobile-apps/features/device-vitals.md]: Done.

File: docs/mobile-apps/features/device-vitals.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
+---
+id: device-vitals
```
[the remark this answers is not in the document]
Comment (aoleszyn) [reply docs/mobile-apps/features/device-vitals.md:2]: As per our slack conversation- resolving the comment :)

File: docs/mobile-apps/features/device-vitals.md (hunk 5 above)
Comment (aoleszyn) [reply docs/mobile-apps/features/device-vitals.md:3]: As per our slack conversation- resolving the comment :)

File: docs/mobile-apps/features/device-vitals.md (hunk 6 above)
Comment (aoleszyn) [reply docs/mobile-apps/features/device-vitals.md:4]: As per our slack conversation- resolving the comment :)

File: docs/mobile-apps/features/device-vitals.md (hunk 2 above)
Comment (lysannep) [inline docs/mobile-apps/features/device-vitals.md]:
```suggestion
Device Vitals is a functionality that allows you to debug and observe an app’s performance during test execution for both [Live](/mobile-apps/live-testing/live-mobile-app-testing/) and [Automated](/mobile-apps/automated-testing/) testing. With this feature, the app's performance is recorded during the test session and displayed for your test results.
```

File: docs/mobile-apps/features/device-vitals.md [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+import TabItem from '@theme/TabItem';
+
+Device Vitals is a functionality that gives you the ability to debug and observe a mobile app’s performance during test execution for both, [Live](/mobile-apps/live-testing/live-mobile-app-testing/) and [Automated](/mobile-apps/automated-testing/) testing. With this feature, the app's performance is recorded on our side during the test session and displayed for your test results.
+
+## What You'll Need
+
+* A Sauce Labs account ([Log in](https://accounts.saucelabs.com/am/XUI/#login/) or sign up for a [free trial license](https://saucelabs.com/sign-up))
+* A native Android, iOS, or iPadOS mobile app
+  
+## Using Device Vitals on Real Devices
```
Comment (lysannep) [inline docs/mobile-apps/features/device-vitals.md:17]:
```suggestion
## Enabling Device Vitals for Real Devices
```

File: docs/mobile-apps/features/device-vitals.md [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+
+Device Vitals is enabled by default if you have the instrumentation feature enabled.
```
Comment (lysannep) [inline docs/mobile-apps/features/device-vitals.md:19]:
```suggestion
By default, Device Vitals is enabled if the instrumentation feature is enabled.
```

File: docs/mobile-apps/features/device-vitals.md [hunk 10]
```diff
@@ -0,0 +1,42 @@
...
+If Instrumentation is disabled, follow the steps below to enable it:
```
Comment (lysannep) [inline docs/mobile-apps/features/device-vitals.md:20]: To enable device instrumentation:

Comment (lysannep) [inline docs/mobile-apps/features/device-vitals.md:20]:
```suggestion
To enable device instrumentation:
```

File: docs/mobile-apps/features/device-vitals.md [hunk 11]
```diff
@@ -0,0 +1,42 @@
...
+Device Vitals is enabled by default if you have the instrumentation feature enabled.
+If Instrumentation is disabled, follow the steps below to enable it: 
+
+1. On Sauce Labs, click **Live** > **Mobile App**.
+2. [Upload your mobile app to Sauce Labs](/mobile-apps/app-storage/) through the UI or CLI.
+3. After you’ve uploaded your app, return to the **Live** > **Mobile App** page, hover your mouse over your app, then select **Settings**. <br/><img src={useBaseUrl('img/mobile-apps/networkcapturescr.png')} alt="Mobile app settings navigation" width="600"/>
+4. Under **Default Settings**, toggle **Instrumentation** on. 
+   <br/><img src={useBaseUrl('img/mobile-apps/Device-Vitals-screenshot-1.png')} alt="Mobile app settings navigation" width="600"/>
+
+Now you can start your live or automated testing session. Your Device Vitals will be captured and be part of your test results page.
```
Comment (lysannep) [inline docs/mobile-apps/features/device-vitals.md:28]:
```suggestion
Now you can start your live or automated testing session. Your Device Vitals will be captured and be part of your test results. 
```

File: docs/mobile-apps/features/device-vitals.md [hunk 12]
```diff
@@ -0,0 +1,42 @@
...
+
+1. On Sauce Labs, click **Live** > **Mobile App**.
+2. [Upload your mobile app to Sauce Labs](/mobile-apps/app-storage/) through the UI or CLI.
+3. After you’ve uploaded your app, return to the **Live** > **Mobile App** page, hover your mouse over your app, then select **Settings**. <br/><img src={useBaseUrl('img/mobile-apps/networkcapturescr.png')} alt="Mobile app settings navigation" width="600"/>
+4. Under **Default Settings**, toggle **Instrumentation** on. 
+   <br/><img src={useBaseUrl('img/mobile-apps/Device-Vitals-screenshot-1.png')} alt="Mobile app settings navigation" width="600"/>
+
+Now you can start your live or automated testing session. Your Device Vitals will be captured and be part of your test results page. 
+
+## Accessing the App Performance metrics
```
Comment (lysannep) [inline docs/mobile-apps/features/device-vitals.md:30]:
```suggestion for consistency
## Accessing Device Vitals Metrics
```

File: docs/mobile-apps/features/device-vitals.md [hunk 13]
```diff
@@ -0,0 +1,46 @@
...
+2. [Upload your mobile app to Sauce Labs](/mobile-apps/app-storage/) through the UI or CLI.
+3. After you’ve uploaded your app, return to the **Live** > **Mobile App** page, hover your mouse over your app, then select **Settings**. <br/><img src={useBaseUrl('img/mobile-apps/networkcapturescr.png')} alt="Mobile app settings navigation" width="600"/>
+4. Under **Default Settings**, toggle **Instrumentation** on. 
+   <br/><img src={useBaseUrl('img/mobile-apps/Device-Vitals-screenshot-1.png')} alt="Mobile app settings navigation" width="600"/>
+
+Now you can start your live or automated testing session. Your Device Vitals will be captured and be part of your test results. 
+
+## Accessing Device Vitals Metrics
+
+The Device Vitals recording will be included in the test results page:
```
Comment (lysannep) [inline docs/mobile-apps/features/device-vitals.md:32]:
```suggestion
You can access Device Vitals metrics from the Test Results page:
```

File: docs/mobile-apps/features/device-vitals.md (hunk 2 above)
Comment (aoleszyn) [reply docs/mobile-apps/features/device-vitals.md]: Done.

File: docs/mobile-apps/features/device-vitals.md [hunk 14]
```diff
@@ -0,0 +1,46 @@
...
+1. On Sauce Labs, click **Live** > **Mobile App**.
+2. [Upload your mobile app to Sauce Labs](/mobile-apps/app-storage/) through the UI or CLI.
+3. After you’ve uploaded your app, return to the **Live** > **Mobile App** page, hover your mouse over your app, then select **Settings**. <br/><img src={useBaseUrl('img/mobile-apps/networkcapturescr.png')} alt="Mobile app settings navigation" width="600"/>
+4. Under **Default Settings**, toggle **Instrumentation** on. 
+   <br/><img src={useBaseUrl('img/mobile-apps/Device-Vitals-screenshot-1.png')} alt="Mobile app settings navigation" width="600"/>
+
+Now you can start your live or automated testing session. Your Device Vitals will be captured and be part of your test results. 
+
+## Accessing Device Vitals Metrics
+
```
Comment (lysannep) [inline docs/mobile-apps/features/device-vitals.md:31]: As per Kris's comment, we should describe the metrics that are captured.

File: docs/mobile-apps/features/device-vitals.md (hunk 2 above)
Comment (lysannep) [inline docs/mobile-apps/features/device-vitals.md]:
```suggestion
Device Vitals is a functionality that allows you to debug and observe an app’s performance during test execution for both [Live](/mobile-apps/live-testing/live-mobile-app-testing/) and [Automated](/mobile-apps/automated-testing/) testing. With this feature, the app's performance is recorded during the test session and displayed for your test results.
```

**Review by lysannep — CHANGES_REQUESTED** [review]
Reviewed

File: docs/mobile-apps/features/device-vitals.md (hunk 14 above)
Comment (linafarid) [reply docs/mobile-apps/features/device-vitals.md:31]: yes, that would be important to describe the data points we show and give an example of their value.
This is from the sales enablement slides perhaps you want to use them:

CPU Performance: 
Detect CPU leaks and over-usage which leads to performance problems, overheating, battery drains, and crashes.

Memory Performance: 
Understand how much memory is actually used by your app. 
Track where you need to reduce your memory footprint to reduce memory leaks. 
Utilize charts to identify where outliers are happening.

UI Responsiveness: 
Measure the delay in UI-responsiveness of apps under test.
Eg: When clicking a CTA button it takes a few seconds to trigger an action and load a new page.

File: docs/mobile-apps/features/device-vitals.md (hunk 4 above)
Comment (aoleszyn) [reply docs/mobile-apps/features/device-vitals.md]: Done.

File: docs/mobile-apps/features/device-vitals.md (hunk 14 above)
Comment (aoleszyn) [reply docs/mobile-apps/features/device-vitals.md:54]: The metrics section has been added.

File: docs/mobile-apps/features/device-vitals.md [hunk 15]
```diff
@@ -0,0 +1,71 @@
...
+<span className="sauceDBlue">Android Only</span>
+
+The Responsiveness metric allows you to quickly assess the app’s health by tracking how long it takes for an app to respond to action and monitoring the delay of the app’s UI responsiveness. Currently, this metric is only available for Android devices.
+
+<br/><img src={useBaseUrl('img/mobile-apps/Device-Vitals-screenshot-6.png')} alt="Mobile app settings navigation" width="600"/>
+
+## Accessing Device Vitals Metrics
+
+You can access Device Vitals metrics from the Test Results page:
+<br/><img src={useBaseUrl('img/mobile-apps/DEvice-Vitals-screenshot-2.png')} alt="Mobile app settings navigation" width="200"/>
```
Comment (kristofmuhi) [inline docs/mobile-apps/features/device-vitals.md:58]: Can we add a highlight for Automated tests too, currently this is just Livetesting highlighted, but most of the value will be for Automated

Comment (kristofmuhi) [reply docs/mobile-apps/features/device-vitals.md:58]: and call it out Automated and Live Test results page

Comment (aoleszyn) [reply docs/mobile-apps/features/device-vitals.md:58]: Automated testing highlighted.

**Review by lysannep — APPROVED** [review]

File: docs/mobile-apps/features/device-vitals.md [hunk 16]
```diff
@@ -0,0 +1,72 @@
...
+<br/><img src={useBaseUrl('img/mobile-apps/DEvice-Vitals-screenshot-7.png')} alt="Mobile app settings navigation" width="200"/>
+
+You will be able to compare your results and analyze the app performance:
+<br/><img src={useBaseUrl('img/mobile-apps/Device-Vitals-Screenshot-3.png')} alt="Mobile app settings navigation" width="800"/>
+
+## Limitations
+
+:::note 
+* Emulators and Simulators are not yet supported.
```
Comment (kristofmuhi) [inline docs/mobile-apps/features/device-vitals.md:67]: lets add not support. the yet part is questionable :D i dont think we will do it in the next 1 year.

File: docs/mobile-apps/features/device-vitals.md [hunk 17]
```diff
@@ -0,0 +1,72 @@
...
+<br/><img src={useBaseUrl('img/mobile-apps/DEvice-Vitals-screenshot-7.png')} alt="Mobile app settings navigation" width="200"/>
+
+You will be able to compare your results and analyze the app performance:
+<br/><img src={useBaseUrl('img/mobile-apps/Device-Vitals-Screenshot-3.png')} alt="Mobile app settings navigation" width="800"/>
+
+## Limitations
+
+:::note 
+* Emulators and Simulators are not yet supported. 
+* Cross-browser testing is not yet available.
```
Comment (kristofmuhi) [inline docs/mobile-apps/features/device-vitals.md:68]: same as the above comment

File: docs/mobile-apps/features/device-vitals.md (hunk 16 above)
Comment (aoleszyn) [reply docs/mobile-apps/features/device-vitals.md]: Corrected :)

File: docs/mobile-apps/features/device-vitals.md (hunk 17 above)
Comment (aoleszyn) [reply docs/mobile-apps/features/device-vitals.md]: Corrected.

**Pull request merged**